standard/expressions.md

```diff
@@ -6195,6 +6213,50 @@ When a property or indexer declared in a *struct_type* is the target of an assig
 >
 > *end example*
 
+### §ref-assignment-new-clause Ref assignment
```

Comment:
Note to update before merging:  Once #742 is merged, the new clause for `ref` safety should link to this location.

Reply:
It was already referenced. I moved it up 3 paragraphs.